tools/run_tests/helper_scripts/build_cxx.sh

```diff
@@ -15,13 +15,32 @@
 
 set -ex
 
+# Set install path to avoid installing to system paths
 cd "$(dirname "$0")/../../.."
+mkdir -p cmake/install
+INSTALL_PATH="$(pwd)/cmake/install"
 
+# Install abseil-cpp since opentelemetry CMake uses find_package to find it.
+cd third_party/abseil-cpp
+mkdir build
+cd build
+cmake -DCMAKE_CXX_STANDARD=14 -DABSL_BUILD_TESTING=OFF -DCMAKE_BUILD_TYPE="${MSBUILD_CONFIG}" -DCMAKE_INSTALL_PREFIX="${INSTALL_PATH}" "$@" ..
```

Comment:
Is it possible to omit this option for all cmake invocations?

Reply:
which option?

Reply:
The `-DCMAKE_CXX_STANDARD=14` is needed for MacOS builds